src/features/classification/security-controls/utils.js

```diff
@@ -76,6 +81,16 @@ const getSharedLinkMessages = (controls: Controls): Array<MessageItem> => {
     return items;
 };
 
+const getWatermarkingMessages = (controls: Controls): Array<MessageItem> => {
+    const items = [];
+    const isWatermarkEnabled = getProp(controls, `${WATERMARK}.enabled`);
```

Comment:
nit: Might be good to add a default `false` value, in case this object is missing

Reply:
updated